Shared/Samples/Download preplanned map area/DownloadPreplannedMapAreaView.Model.swift

```diff
@@ -299,12 +299,12 @@ private extension OfflineMapModel {
 
 private extension FileManager {
     /// Creates a temporary directory and returns the URL of the created directory.
-    static func createTemporaryDirectory() -> URL {
+    static func makeTemporaryDirectory() -> URL {
```

Comment:
We went with `create` because this method creates something (a directory on disk) rather than makes something (a value of some type). The API design guideline doesn't apply here because this isn't a factory method. The old name is more appropriate.